integration/webhooks.rst

```diff
@@ -0,0 +1,80 @@
+.. _webhooks:
+
+Webhooks
+========
+
+Platform.sh can easily be integrated with external services using webhooks.
+
+With the CLI, you can list all your active integrations:
+
+.. code-block:: console
+
+    $ platform integration:list
```

Comment:
I think we shouldn't include the `$` sign in code examples, it makes it harder to copy/paste